pkgs/games/quakespasm/vulkan.nix

```
@@ -34,19 +54,16 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/bin"
   '';
 
-  makeFlags = [ "prefix=$(out) bindir=$(out)/bin" ];
-
   env = lib.optionalAttrs stdenv.isDarwin {
     NIX_CFLAGS_COMPILE = "-Wno-error=unused-but-set-variable";
   };
 
   postFixup = ''
+    cp vkquake "$out/bin"
     wrapProgram $out/bin/vkquake \
       --prefix LD_LIBRARY_PATH : ${vulkan-loader}/lib
```

```suggestion
    patchelf $out/bin/vkquake \
      --add-rpath ${lib.makeLibraryPath [ vulkan-loader ]}
```
we can avoid setting LD_LIBRARY_PATH